src/utils/MessageBuilder.ts

```diff
@@ -1,15 +1,25 @@
 import { ButtonInteraction, MessageActionRow, MessageButton, MessageEmbed, MessageOptions } from "discord.js";
+import { PlayerInActiveMatch } from "../repositories/ActiveMatchRepository/types";
+import { Team } from "../types/common";
+import getEnvVariable from "./getEnvVariable";
 import { PlayerInQueue } from "../repositories/QueueRepository/types";
 
 export const enum ButtonCustomID {
   JoinQueue = "joinQueue",
   LeaveQueue = "leaveQueue",
+  CreateRandomTeam = "randomizeTeams",
+  FillTeam = "fillTeam",
+  ReportMatch = "reportMatch",
+  RemoveAll = "removeAll",
+  BreakMatch = "breakMatch",
 }
 
 export default class MessageBuilder {
   private static readonly normIconURL =
     "https://raw.githubusercontent.com/mattwells19/UNCC-Six-Mans.js/main/media/norm_still.png";
 
+  private static readonly isDev = getEnvVariable("ENVIRONMENT") == "dev" ? true : false;
```

Comment:
Use `===` for strict comparison. And you can remove the `? true : false` since it's already evaluating to a boolean.

Reply:
Fixed in next commit.